src/envoy/http/jwt_auth/pubkey_cache.h

```diff
@@ -46,6 +49,17 @@ class PubkeyCacheItem {
     for (const auto& aud : jwt_config_.audiences()) {
       audiences_.insert(SanitizeAudience(aud));
     }
+
+    auto inline_jwks = Config::DataSource::read(jwt_config_.local_jwks(), true);
+    if (!inline_jwks.empty()) {
+      Status status = SetKey(Base64::decode(inline_jwks),
```

Comment:
Not Base64::decode?

Reply:
I think it is still better to base64 encod it.  Here is my concern, Envoy is using JsonConvert() to convert Struct or Protobuf to other config proto.   If we don't base64 encode jwks,  jwks is a JSON,  it will break.  It is safer to base64 encode the jwks.

Reply:
No please dont, it won't break JSON encoding, a string will be escaped correctly. If you want base64 encoded one, just use inline_bytes, which is by proto spec. You don't want to load base64 encoded from DataSource file. please revert.